lib/scripts/users_list.rb

```diff
@@ -0,0 +1,32 @@
+require 'csv'
+
+class UsersList
+  attr_accessor :csv_file
+
+  def initialize
+    @csv_file = CSV.open(File.join(Rails.root, "/lib/scripts/results/users_list.csv"), "w") do |csv|
+      csv << ['Formal Name', 'netID', 'email', 'Current sign in', 'Last sign in']
+    end
+  end
+
+  def populate_user_info
+    if self.csv_file.closed?
+      self.csv_file.reopen(self.csv_file.path, "a")
+    end
+
+    select_users.find_in_batches do |users_batch|
+      users_batch.each do |user|
+        self.csv_file << [user.formal_name, user.username, user.email,
+                          user.current_sign_in_at, user.last_sign_in_at]
+      end
+    end
+
+    self.csv_file.close
+  end
+
+  private
+  def select_users
```

Comment:
This is nitpicking, but I think for clarity we should rename this to `select_all_users` and/or update the function body to use `User.all.select(...)`. Personally I had to run some code locally to confirm what `User.select` would return here.

Reply:
Nitpicking is fine, as long it makes it more readable.